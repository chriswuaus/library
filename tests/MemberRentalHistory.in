MEMBER HISTORY 100000
add member beanboy bean
member history 1010101
member history 100000
add collection test.csv
rent 100000 111111
member history 100000
relinquish 100000 111111
member history 100000
rent 100000 111111
rent 100000 111112
rent 100000 111113
rent 100000 111114
rent 100000 111115
rent 100000 111116
rent 100000 111117
rent 100000 111118
member history 100000
relinquish 100000 111111
relinquish 100000 111112
relinquish 100000 111113
relinquish 100000 111114
relinquish 100000 111115
relinquish 100000 111116
relinquish 100000 111117
relinquish 100000 111118
member history 100000
exit
