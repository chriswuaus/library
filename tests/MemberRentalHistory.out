user: No members in system.

user: Success.

user: No such member in system.

user: No rental history for member.

user: 17 books successfully added.

user: Success.

user: No rental history for member.

user: Success.

user: To Kill a Mockingbird (Harper Lee)

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: To Kill a Mockingbird (Harper Lee)

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: To Kill a Mockingbird (Harper Lee)
To Kill a Mockingbird (Harper Lee)
The Hobbit (J.R.R. Tolkien)
The Iliad (Homer)
The Odyssey (Homer)
The Hitchhiker's Guide to the Galaxy (Douglas Adams)
So Long and Thanks for all the Fish (Douglas Adams)
Batman Volume 1 (Bob Kane)
Harry Potter 1 (J.K. Rowling)

user: Ending Library process.
